libbuild2-autoconf/libbuild2/autoconf/checks/HAVE_ARC4RANDOM_BUF.h

```diff
@@ -0,0 +1,17 @@
+// HAVE_ARC4RANDOM_BUF : BUILD2_AUTOCONF_LIBC_VERSION
+
+#ifndef BUILD2_AUTOCONF_LIBC_VERSION
+#  error BUILD2_AUTOCONF_LIBC_VERSION appears to be conditionally included
+#endif
+
+#undef HAVE_ARC4RANDOM_BUF
+
+/* Since OpenBSD 2.1, FreeBSD 2.3, NetBSD 2.0, and Mac OS X (10.7 according 
+ * to comment in libevent)
+ */
+#if BUILD2_AUTOCONF_OPENBSD_PREREQ(199706) || \
```

Comment:
`arc4random_buf `has been added in the latest glibc `2.36`
`BUILD2_AUTOCONF_GLIBC_PREREQ(2, 36)`
see https://www.phoronix.com/news/GNU-Glibc-arc4random-Functions and 
https://elixir.bootlin.com/glibc/glibc-2.36/source/stdlib/stdlib.h